samples/cpp/benchmark_app/CMakeLists.txt

```diff
@@ -50,10 +50,10 @@ if(OpenCL_FOUND AND OpenCL_HEADERS)
 endif()
 
 # Optional OpenCV dependnency
-
 find_package(OpenCV COMPONENTS core QUIET)
 if(NOT OpenCV_FOUND)
     message(WARNING "OpenCV is disabled or not found, ${TARGET_NAME} will be built without OpenCV support. Set OpenCV_DIR")
+    target_link_libraries(${TARGET_NAME} PRIVATE nlohmann_json_schema_validator)
```

Comment:
I think we also need to add lines like :
https://github.com/openvinotoolkit/openvino/blob/8453b6395a447e897c7f3a4a7a0deeb38447a606/samples/cpp/CMakeLists.txt#L118-L120
to `openvino/samples/cpp/CMakeLists.txt`

Reply:
thanks!
done